.github/workflows/sdk-release-repo-branch-check.yml

```diff
@@ -1,25 +1,19 @@
-name: "Semantic Release Repo & Branch Check"
+name: "Semantic Release Branch Check"
 on:
   workflow_call:
 jobs:
-  # SDK release must be done from the public/master or public/main branch
+  # SDK release must be done from the master or main branch
   confirm-public-repo-master-branch:
-    name: Confirm release is run on public/master or public/main branch
+    name: Confirm release is run on master or main branch
```

Comment:
is it worh updating the job name to be `confirm-master-main-branch`, removing the `public`?

Reply:
or are there workflows that are referencing this job name?  (I'm not an expert in how these common workflows are used, but I think it's safe the change the name...

Reply:
if we change the name wouldn't we have to update that everywhere the shared workflow is used?

Reply:
I checked the workflows here https://github.com/mParticle/mparticle-apple-sdk/tree/main/.github/workflows and don't see mention of job names, and only the workflow file name...so I think it's safe to remove.

Reply:
The workflow is accessed by the yml file name:

```
# SDK release is done from main branch.
  confirm-main-branch:
    name: Confirm release is run from main branch
    uses: mParticle/mparticle-workflows/.github/workflows/sdk-release-repo-branch-check.yml@stable
 ```
 
 So it should be fine to change the action name as long as we don't rename the file